### ISSUE TITLE
`incomplete-signature-doc` does not appear to work

### ISSUE THREAD

**Issue by lewis6991** [EmmyLuaLs/emmylua-analyzer-rust#592]
```lua
--- @param a integer
local function foo(a)
  return a + 1
end
```

config:

```json
{
  "$schema": "https://raw.githubusercontent.com/EmmyLuaLs/emmylua-analyzer-rust/refs/heads/main/crates/emmylua_code_analysis/resources/schema.json",
  "runtime": {
    "version": "LuaJIT"
  },
  "diagnostics" : {
    "disable" : [
      "unnecessary-if"
    ],
    "enables": [
      "incomplete-signature-doc",
      "missing-global-doc"
    ]
  }
}
```

Should get warning about missing `@return`

**Comment by lewis6991** [comment]
Working now. Must have done something wrong.
